WorkingWithFiles/Program.cs

```diff
@@ -17,19 +17,67 @@ public static class WorkingWithFiles
         // .Trim() might be useful in this situation.
         public static int WordCount(string fileName)
         {
-            return default;
-        }
+            var streamReader = new StreamReader(fileName);
+            var count = 0;
+            string[] words = null;
+            string sentence = null;
 
+            while (!streamReader.EndOfStream)
+            {
+                sentence = streamReader.ReadLine().Trim();
+                words = sentence.Split(' ');
+                count += words.Length;
+            }
+            return count;
+        }
         // 2- Write a method that reads a text file and returns the longest word in the file. Ex.
         // The file contains, "The dog named Maximus ran away from home." The method should return
         // "Maximus" the longest word contained in the file. Return "File is Empty" if the file is empty.
         // .Trim() might be useful in this situation.
         public static string LongestWord(string fileName)
         {
-            return default;
+            var streamReader = new StreamReader(fileName);
+            string[] words = null;
+            string sentence = null;
+            string longestWord = "File is Empty";
+
+            while (!streamReader.EndOfStream)
+            {
+                sentence = streamReader.ReadLine().Trim();
+                words = sentence.Split(' ');
+
+                var beginning = words[0];
+                for (var i = 0; i < words.Length; i++)
+                {
+
+
+                    if (words[i].Length > beginning.Length)
+                    {
+                        longestWord = words[i];
+
+                    }
+                    else
+                    {
+                        longestWord = beginning;
+                    }
+                }
+
+                string Longestword = beginning;
+
+                while (!streamReader.EndOfStream)
```

Comment:
I haven't seen this yet as a double embedded while loop. I'm actually surprised it works 😮